Assets/MixedRealityToolkit/StandardAssets/Shaders/MixedRealityStandard.shader

```diff
@@ -63,6 +63,8 @@ Shader "Mixed Reality Toolkit/Standard"
         [Toggle(_ROUND_CORNERS)] _RoundCorners("Round Corners", Float) = 0.0
         _RoundCornerRadius("Round Corner Radius", Range(0.0, 0.5)) = 0.25
         _RoundCornerMargin("Round Corner Margin", Range(0.0, 0.5)) = 0.01
+		[Toggle(_INDEPENDENT_CORNERS)] _IndependentCorners("Independent Corners", Float) = 0.0
```

Comment:
The rest of this script looks like it uses spaces instead of tabs, which makes the spacing on these lines look odd. Please replace the tabs with spaces.

![image](https://user-images.githubusercontent.com/3580640/71842435-7bd43380-3076-11ea-9d16-85f9dfa3a2dd.png)